test/03_USM.test.js

```diff
@@ -23,69 +24,74 @@ contract('USM', (accounts) => {
     beforeEach(async () => {
       // Deploy contracts
       oracle = await TestOracle.new(price, shift, { from: deployer })
-      usm = await USM.new(oracle.address, { from: deployer })
+      weth = await WETH9.new({ from: deployer })
+      usm = await USM.new(oracle.address, weth.address, { from: deployer })
       fum = await FUM.at(await usm.fum())
+
+      // Set the user to obtain 10 Weth and approve USM to take them.
```

Comment:
The user needs to deposit Ether into the Weth contract to get Weth. Then he needs to approve USM as usual, for the ERC20 transfers to happen.